src/client.coffee

```diff
@@ -20,17 +20,23 @@ parser = new xml2js.Parser
     default_host {String} Optional
   Example:
     Client = require('3scale').Client
-    client = new Client(provider_key, [default_host])
+    client = new Client(provider_key, [default_host], [default_port])
 ###
 
+
 module.exports = class Client
   DEFAULT_HEADERS: { "X-3scale-User-Agent": "plugin-node-v#{VERSION}" }
 
-  constructor: (provider_key, default_host = "su1.3scale.net") ->
+  constructor: (provider_key, provider_host, provider_port) ->
```

Comment:
Nitpick: I think that `provider_host` and `provider_port` are misleading name vars. They are referring to 3scale's service management API host and port, not to anything related to the provider. I think that in this case using `host` and `port` is fine.